webpack.config.js

```diff
@@ -12,20 +11,12 @@ module.exports = {
   },
   resolve: {
     alias: {
-      'react-native': join(__dirname, '/mocks/react-native'),
+      'react-native$': join(__dirname, 'mocks/react-native'),
     },
   },
   module: {
     rules: [
       { test: /\.js$/, exclude: /node_modules/, use: ['babel-loader'] },
     ],
   },
-  plugins: [
-    new webpack.DefinePlugin({
-      'process.env': {
-        NODE_ENV: JSON.stringify('production'),
-      },
-    }),
-    new webpack.optimize.UglifyJsPlugin(),
```

Comment:
I think these plugins do actually affect performance—especially the first! Should we leave them in for more accurate results?